src/main/java/com/gluonhq/substrate/model/ClassPath.java

```diff
@@ -28,7 +28,12 @@
      * @param classPath standard java classpath, delimited with {@code File.pathSeparator}. Should not be null.
      */
     public ClassPath(String classPath) {
-        this.classPath = Objects.requireNonNull(classPath);
+        String rawClassPath = Objects.requireNonNull(classPath);
```

Comment:
Is it possible that the classpath contains something like:
`"path1.to.jar1.jar";"path2.to.jar2.jar";...;"pathn.to.jarn.jar"`?
If that is the case, removing first and last quote might work, but `asStream()` might fail?

Reply:
On Windows, the value of the `-cp` argument for `native-image.cmd` needs to be wrapped in double quotes. I'll update the code, so that these quotes are only added at the very end, when actually building the process of the native-image command. That way, we won't need to deal with it when working with the classpath in other sections of the code.